parts/inc/Sv_set

```diff
@@ -101,10 +103,24 @@ __UNDEFINED__ sv_setsv_flags(dstr, sstr, flags)
   )
 #endif
 
-#if defined(PERL_USE_GCC_BRACE_GROUPS)
-__UNDEFINED__ newSVsv_flags(sv, flags) ({ SV *_sv = newSV(0); sv_setsv_flags(_sv, (sv), (flags)); _sv; })
-#else
-__UNDEFINED__ newSVsv_flags(sv, flags) ((PL_Sv = newSV(0)), sv_setsv_flags(PL_Sv, (sv), (flags)), PL_Sv)
+#ifndef newSVsv_flags
+#  if defined(PERL_USE_GCC_BRACE_GROUPS)
+#    define  newSVsv_flags(sv, flags)                       \
+        ({                                                  \
+            SV *new = newSV(0);                             \
+            sv_setsv_flags(new, (sv), (flags));             \
+            new;                                            \
```

Comment:
variable name  change here with multiline indentation

Reply:
I don't understand your point here.  I can add something to the commit message about these; but without the name change you have two variables that look almost alike.

Reply:
this was just a comment regarding the `new` nothing to do there